accounts/checkers.go

```diff
@@ -531,6 +532,8 @@ func checkSend(ctx context.Context, chainParams *chaincfg.Params,
 		if payReq.MilliSat != nil && *payReq.MilliSat > sendAmt {
 			sendAmt = *payReq.MilliSat
 		}
+
+		paymentHash = *payReq.PaymentHash
```

Comment:
Throw a `nil` check in here, just in case? Not sure if an AMP invoice really has a payment hash in every case.